spinalcordtoolbox/reports/qc.py

```diff
@@ -413,4 +419,13 @@ def normalized(img):
 
     sct.printv('Sucessfully generated the QC results in %s' % qc_param.qc_results)
     sct.printv('Use the following command to see the results in a browser:')
-    sct.printv('open file "{}/index.html"'.format(path_qc), type='info')
+    try:
+        from sys import platform as _platform
+        if _platform == "linux" or _platform == "linux2":
+            sct.printv('xdg-open "{}/index.html"'.format(path_qc), type='info')
+        elif _platform == "darwin":
+            sct.printv('open "{}/index.html"'.format(path_qc), type='info')
+        else:
```

Comment:
We could bail out here (`RuntimeError`). Then we will remember to fix that when we  want to support another platform.

Reply:
(ImportError, RuntimeError) would do the job?